src/specs/api.yaml

```diff
@@ -405,16 +405,109 @@ paths:
           content:
             application/json:
               schema:
                 $ref: ./models/SampleResponse.v1.yaml
-        "400":
+        '400':
           description: Samples not found.
           content:
             application/json:
               schema:
                 $ref: ./models/HTTPError.v1.yaml
       description: Get all samples for an experiment
+  '/projects/{projectUuid}':
+    parameters:
+      - schema:
+          type: string
+        name: projectUuid
+        in: path
+        required: true
+    put:
+      summary: Create or update project
+      operationId: updateProject
+      x-eov-operation-id: projects#update
+      x-eov-operation-handler: routes/projects
+      responses:
+        '200':
+          description: Create or update sample object
+          content:
+            application/json:
+              schema:
+                $ref: ./models/api-body-schemas/Project.v1.yaml
+        '201':
+          description: New resource created.
+          content:
+            application/json:
+              schema:
+                $ref: ./models/api-body-schemas/Project.v1.yaml
+        '415':
+          description: Unsupported Media Type
+        '404':
+          description: Invalid projectUuid.
+      description: Create or update a project
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: ./models/api-body-schemas/Project.v1.yaml
+        description: The new configuration to update the old one by.
+  '/projects/{projectUuid}/samples':
+    parameters:
+      - schema:
+          type: string
+        name: projectUuid
+        in: path
+        required: true
+    put:
+      summary: Update sample
+      operationId: updateSample
+      x-eov-operation-id: samples#update
+      x-eov-operation-handler: routes/samples
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties: {}
+        '415':
+          description: Unsupported Media Type
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties: {}
+      description: Updates a sample
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                projectUuid:
+                  type: string
+                experimentId:
+                  type: string
+                samples:
+                  $ref: ./models/api-body-schemas/Samples.v1.yaml
+    get:
+      summary: 'Get samples under a project '
+      operationId: getProjectSamples
+      x-eov-operation-id: samples#get
+      x-eov-operation-handler: routes/samples
+      responses:
+        '200':
```

Comment:
the response can also be 404 if the sample is not found

Reply:
Ok, will return this